src/Microsoft.Health.Fhir.SqlServer/Features/Schema/Migrations/10.diff.sql

```diff
@@ -0,0 +1,585 @@
+SET NOCOUNT ON
```

Comment:
>SET [](http://example.com/codeflow?start=0&length=3)

I can probably do comparison between 9 and 10 version, and then between 10 and 11, and try to deduct why 2 versions needed, but I better ask directly.

Most of the time it's one time migration per PR, why it's double migration this time?

Reply:
First one ensures that new data will have the new column set (the stored procedure writes is). The second one updates all the existing data. When the second is done, you can start reading from the column.

Reply:
Actually, I guess you could combine them.